eos-prune-printers

```diff
@@ -36,9 +36,22 @@ done
 systemctl stop cups
 
 # remove stamp file from eos-config-printer (from 4.0 use driverless printers)
-rm -rf /var/lib/eos-config-printer
+LIB_CONFIG_PRINTER=/var/lib/eos-config-printer
+if [ -d $LIB_CONFIG_PRINTER ]; then
+	echo Removing $LIB_CONFIG_PRINTER
+	rm -rf $LIB_CONFIG_PRINTER
+else
+	echo $LIB_CONFIG_PRINTER does not exist
+fi
+
 # clean up the residual print cache files
-rm -rf /var/cache/cups/*
+CACHE_CUPS=/var/cache/cups
+if [ -d $CAHCE_CUPS ]; then
+	echo Removing ${CACHE_CUPS:?}/*
```

Comment:
You followed the shellcheck warning but I don't think this does what you hope for.

```
       ${parameter:?word}
              Display Error if Null or Unset.  If parameter is null or unset, the expansion of word (or a message to that effect
              if  word  is not present) is written to the standard error and the shell, if it is not interactive, exits.  Other‐
              wise, the value of parameter is substituted.
```

CACHE_CUPS is always set so I don't think this or the line below do what you hope.

```
wjt@camille:~$ mkdir /tmp/foo
wjt@camille:~$ X=/tmp/foo
wjt@camille:~$ echo Removing $X/*
Removing /tmp/foo/*
wjt@camille:~$ echo Removing ${X:?}/*
Removing /tmp/foo/*
wjt@camille:~$ touch /tmp/foo/{a,b,c}
wjt@camille:~$ echo Removing $X/*                                                                                               
Removing /tmp/foo/a /tmp/foo/b /tmp/foo/c
wjt@camille:~$ echo Removing ${X:?}/*                                                                                               
Removing /tmp/foo/a /tmp/foo/b /tmp/foo/c
```

Reply:
Seems I missed some meaning in the description.  Thanks